serverless.ts

```diff
@@ -392,12 +377,12 @@ const serverlessConfig: AWS = {
           eventBridge: {
             eventBus: 'asap-events-${self:provider.stage}',
             pattern: {
-              source: ['asap.lab'],
+              source: ['asap.entity-updated'],
```

Comment:
maybe declare `eventSource` somewhere at the top and use here and at #576? same goes for the event bus